src/app/create-event/page.tsx

```diff
@@ -16,23 +16,20 @@ export default function CreateEvent() {
   const [earliestTime, setEarliestTime] = useState('9:00 AM')
   const [latestTime, setLatestTime] = useState('5:00 PM')
   const [mode, setMode] = useState('weekly')
-  const [config, setConfig] = useState<string[]>([])
+  const [config, setConfig] = useState<string[]>([]) // daysOfWeek
```

Comment:
small nit: isn't the comment misleading since it is both daysOfWeek and specific dates.

Reply:
removing this in next commit